.github/workflows/zulip-ci.yml

```diff
@@ -9,6 +9,7 @@ defaults:
 jobs:
   focal_bionic:
     strategy:
+      fail-fast: false
```

Comment:
Can you add a commit message for this change?  I assume the intent is that we want to still run the production builds when the backend ones fail (or vise versa)?

Reply:
No, that's not what this change does. This makes it so other jobs in the same matrix run when one fails. In this case, if focal backend fails then this makes it so bionic does not.